Lesson33/Selcan/lesson33-app/src/Exercise8/index.jsx

```diff
@@ -0,0 +1,54 @@
+//Exercise 8: Dynamic List Rendering (add these buttons to the existing todo list items in our application!)
+// Render a list of items with buttons next to them. Clicking a button should remove the corresponding item from the list.
+   
+import { useState } from "react";
+
+export const Exercise8 = () => {
+  const [items, setItems] = useState(['Item 1', 'Item 2', 'Item 3']);
+
+  const removeItem = (index) => {
+    setItems(prevItems => prevItems.filter((item, i) => i !== index));
```

Comment:
The solution works however relying on the index in the array is not reliable. Normally, in real life we try to use an `id` (unique identifier, UUID is commonly used: https://www.uuidgenerator.net/) so that we have identifier that do not change. 
The data usually looks like this in this case:
```
{id :"c3c6a6c6-b0e1-49a9-8e3e-cbae1d8617cd",
title: "Item 1"
}
```

No need to change anything now, but worth knowing it for the future. :)